test_export.py

```
@@ -0,0 +1,161 @@
+        timestamp=datetime.utcnow(),
+        versao_analise="2.1.0"
+    )
+
+    try:
+        # Generate PDF
+        print("📄 Generating PDF report...")
+        pdf_bytes = await report_generator.generate_verification_pdf(resultado)
+
+        # Save to file
+        output_file = "test_relatorio_verificacao.pdf"
+        with open(output_file, 'wb') as f:
```

Ruff (0.14.5)

Async functions should not open files with blocking methods like `open`

(ASYNC230)